example/opensourcecmf/opensource_replica_msgs.cmf

```diff
@@ -0,0 +1,62 @@
+# Definition of client-server interface for the opensource example test application.
+#
+# Note OSRKVBC effectively implements a versioned key-value store supporting
+# only keys and values that are byte strings.
+
+Msg OSRKVBCReadRequest 3 {
```

Comment:
The "OSRKVBC" prefix is a bit cryptic IMO... Assuming "OSR" means open source, I think we can drop it as it is obvious this is an open source project :) Can we simplify the message names further, to something like `KVReadRequest`, etc.?

CC: @arc-vmware